verilog/tools/kythe/indexing_facts_tree_extractor.cc

```diff
@@ -126,6 +126,16 @@ void IndexingFactsTreeExtractor::Visit(const SyntaxTreeNode& node) {
           FindAllVariableDeclarationAssignment(node);
 
       if (!variable_declaration_assign.empty()) {
+        // for classes.
+        const std::vector<TreeSearchMatch> class_instances =
+            verible::SearchSyntaxTree(node, NodekClassNew());
+        if (!class_instances.empty()) {
+          ExtractClassInstances(node, variable_declaration_assign);
+          break;
```

Comment:
Our coverage reports show these lines weren't covered.  Is it easy to add a test case that exercises these?